packages/docusaurus/lib/default-theme/DocBody/index.js

```diff
@@ -9,20 +9,27 @@ import React, {useContext, useEffect} from 'react';
 
 import DocsPaginator from '@theme/DocsPaginator'; // eslint-disable-line
 import DocusaurusContext from '@docusaurus/context';
+import Head from '@docusaurus/Head';
 
 import styles from './styles.module.css';
 
 function DocBody(props) {
   const {metadata, modules} = props;
+  const {language, version} = metadata;
   const context = useContext(DocusaurusContext);
   useEffect(() => {
     context.setContext({metadata});
   }, []);
 
   const DocContents = modules[0];
-
   return (
     <div className={styles.docBody}>
+      <Head>
```

Comment:
In the future I'd like to remove the duplication across files. We seem to have many of these `<Head>` with similar contents around - blog, page, docs, etc.

Reply:
Sure. This <Head> is needed in here because it will override the default parent <Head>.